### PR TITLE
Add Codebase Expectations section to the Contributor Guidelines

### PR DESCRIPTION
**Pull request by ZeWaka**
## About the PR 

adds it :godmode:

*yeah i'm using a label i deprecated. so what? fite me*

## Why's this needed? 

needed

### CODE REVIEW

File: .github/CONTRIBUTOR_GUIDELINES.md [hunk 1]
```diff
@@ -4,6 +4,39 @@
...
+The question this section aims to answer is: "What can I reasonably expect about this codebase, my contributions, and the maintainers?"
+
+Essentially, this is more of a FAQ section.
+
+### Maintainers / Developers
+
+The Goonstation codebase maintainers/developers are a part of the general admin team for the Goonstation servers. They are The People Who Decide What Goes In The Game. You can identify these people on the discord by their red/amaranth name.
+On GitHub, they'll also show up as a member of the Goonstation organization: ![](https://i.imgur.com/xcWNk5p.png).
+
+Please do not unnecessarily ping us directly unless someone explicitly says it's ok. If you have a question for developers, you can generally reach us (or perhaps a fellow contributor can answer your question) in the #imXYZ channels, like #imcoder.
```
Comment (ZeWaka) [inline .github/CONTRIBUTOR_GUIDELINES.md:18]:
```suggestion
Please do not unnecessarily ping us directly unless someone explicitly says it's ok. If you have a question for developers, you can generally reach us (or perhaps a fellow contributor can answer your question) in the #imXYZ discord channels, like #imcoder.
```
clarify

**Comment by DisturbHerb** [comment]
I swear I'll review this at some point, I'm just busy getting over an illness at the moment.

**Comment by ZeWaka** [comment]
x

File: .github/CONTRIBUTOR_GUIDELINES.md [hunk 2]
```diff
@@ -4,6 +4,39 @@
...
 
+## Codebase Expectations
+
+The question this section aims to answer is: "What can I reasonably expect about this codebase, my contributions, and the maintainers?"
+
+Essentially, this is more of a FAQ section.
+
+### Maintainers / Developers
+
+The Goonstation codebase maintainers/developers are a part of the general admin team for the Goonstation servers. They are The People Who Decide What Goes In The Game. You can identify these people on the discord by their red/amaranth name.
```
Comment (flrsh) [inline .github/CONTRIBUTOR_GUIDELINES.md:15]:
```suggestion
The Goonstation codebase maintainers/developers are a part of the general admin team for the Goonstation servers. They are The People Who Decide What Goes In The Game. You can identify these people on the Discord by their red/amaranth name.
```

**Review by flrsh — APPROVED** [review]
Ty for writing this.

**Review by Azrun — APPROVED** [review]

**Comment by ZeWaka** [comment]
Todo: put in articles and links but I can do that later

**Pull request merged**